presto-main-base/src/main/java/com/facebook/presto/server/PluginManager.java

```diff
@@ -506,4 +521,20 @@ private void processPlugins(Artifact artifact, ClassLoader classLoader, String s
             writePluginServices(plugins, artifact.getFile(), servicesFile);
         }
     }
+
+    private boolean loadPluginIfConfigured(Map<String, String> expectedProperties)
```

Comment:
I found the name confusing since this doesn't do the loading. Can we rename it? Maybe something like `pluginIsEnabled`.